arch/lkl/kernel/atomic.c

```diff
@@ -0,0 +1,86 @@
+#include <linux/kernel.h>
```

Comment:
@thehajime I cannot find the code changes and the file called "atomic.c " in my code base, so the code are not merged into the master branch, right?  
Besides the __sync_fetch_xx ops mentioned here, the following symbols ("__sync_fetch_and_or_4", "__sync_fetch_and_and_4", "__sync_fetch_and_add_4" and "__sync_fetch_and_sub_4") are still undefined for ARM 32bit platform.